### ISSUE TITLE
[android] Bicycle Dock stations page info and search metadata

### ISSUE THREAD

**Issue by patepelo** [organicmaps/organicmaps#6657]
Currently, bicycle dock stations are represented like in this example from Barcelona:

![image](https://github.com/organicmaps/organicmaps/assets/23003127/8caa9306-0200-4320-bb3b-83d91f220899)

The problem is that ClearChannel doens't tell a thing to the en user since it is the operator=* and it is a private company with a concession. 

More important is to display the network= --> Bicing in Barcelona's example. 

Based on the [wiki: Bicycle rental](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_rental).

operator=* is optional and network=* is more important. This information should also be available on search, meaning that:

- user searches for "bicing" and gets all the dock station results. 

____

Beyond this, maybe we can use the ref=* instead of the name to be shown on the map, or just the network=*.

The assigned name=* could be shown upon opening of the POI.

____

Instead of bicycle rental, maybe we should make it it's own category as "Dock Station" and starting to differentiate it from other bicycle rentals that will have their own rental shop circled icon in the future.

**Comment by pastk** [comment]
> Instead of bicycle rental, maybe we should make it it's own category as "Dock Station" and starting to differentiate it from other bicycle rentals that will have their own rental shop circled icon in the future.

Yeap, it was discussed here
- https://github.com/organicmaps/organicmaps/pull/6192#discussion_r1346134727

And there was a docking station icon attempt.

**Comment by patepelo** [comment]
Yes, I was making some drafts with the idea of differentiating from other bicycle rental shop.

For docking station, some drafts:
![image](https://github.com/organicmaps/organicmaps/assets/23003127/03c6512f-d361-428b-af74-3cef50741616)


For  shop rental, @zyphlar and I have been exchanging ideas in Figma. Where I used the original icon just circled and @zyphlar  was proposing an alternative with dollar sign, though I should and will open a new issue for this one specifically. 

![image](https://github.com/organicmaps/organicmaps/assets/23003127/15ad419a-3fd5-4f9d-926a-1b60037bbc3c)

**Comment by zyphlar** [comment]
While we're on the topic, I really like the minimalist line-art bicycle in that last purple photo, it really looks like a bicycle. Sometimes I struggle with the bicycle icon with the thicker horizontal lines representing the chain guard because it's so easy to look like a motorcycle, moped, or e-bike. If we can universally standardize on the "upside down Y" shape including in the bicycle route planner, I think that would be a big improvement.

For bicycle dock, the bottom-left icon where you can see the front tire pass through the solid dock feels best to me. Top-left could also work if the dock extended doubly thick to the left so it doesn't look like a bike rack.

**Comment by zyphlar** [comment]
As an American I wouldn't object ;) but I think "bicycle parking" is also a sufficiently clear icon as it exists, and the blue/purple iconography are pretty clear as far as shop + transit infra. Bicycle parking isn't really infra so much as asphalt and metal designated for a purpose. People can get confused looking at a new app at first but once they see how things are distinguished they can usually figure it out, especially if pressing on the icon brings up relevant information.

**Comment by patepelo** [comment]
There are few thing as universal recognized as the dollar sign, I don't see that as the problem. The real thing is how it looks when tiny. The vertical placement makes the bicycle displaced down and it seems as if it would be a person riding a bicycle.

When we post big icons we should:
- zoom out the browser to the minimum size
- close one eye 
- look from 1 meter apart 
Then we could get a better idea of a new user would see it.

Here are the options represented on a screenshot example.
![image](https://github.com/organicmaps/organicmaps/assets/23003127/ff0eaf85-7d3c-4aca-b472-ba18090d6633)

**Comment by biodranik** [comment]
It looks like a Toyota motorbike )

**Comment by biodranik** [comment]
1. What about removing the background for docking stations and reusing space to make the icon bigger?
2. Raw idea: Use a wheel (with pedals) icon for round shops/businesses.

**Comment by VladimirMorozov** [comment]
At first I thought that these icons are for different bicycle parking types. There is a need to have them displayed differently, because they allow different security, so a user may assume that it's a parking type. Also, in Estonia there are free bicycle parking **docks** which are locked by app/phone call/card.

I think dollar would be better and less confusing.

**Comment by Misalf-git** [comment]
> There are few thing as universal recognized as the dollar sign, I don't see that as the problem. The real thing is how it looks when tiny. The vertical placement makes the bicycle displaced down and it seems as if it would be a person riding a bicycle.

I see the dollar-person, too.

**Comment by biodranik** [comment]
There is already a paid parking icon. Maybe reuse a similar idea?
